app/javascript/gobierto_dashboards/webapp/containers/shared/Aside.vue

```diff
@@ -167,6 +185,9 @@ export default {
           _filter.isToggle = !_filter.isToggle;
         }
       })
+    },
+    showDownloadDataLink(){
+      return(this.dataDownloadEndpoint != '' && this.dataDownloadEndpoint != undefined)
```

Comment:
1. ejem ejem.... `!==`
2. Look what are you doing: firstly, you check if not empty, then check if not undefined. That's nonsense. If the first condition pass, the second one also will. If you'd have to, you'd have to do it backwards
3. Coolest way: `return !!this.dataDownloadEndpoint`